docs/reference/2-ods-api/platform-dev-guide/features/changed-record-queries.md

```diff
@@ -219,7 +219,40 @@ of the sequence on all inserts and updates, whether they come from the API or
 through scripts. Queries done against the API to find changed records function
 by adding an additional where clause to based on this ChangeVersion column.
 
-![Change Queries](../../../../../static/img/reference/ods-api/image2022-5-11_10-27-25.png)
+```mermaid
+
+%%{init: {
+    "theme": "",
+    "themeCSS": [
+        "[id*=entity-Student] rect:nth-of-type(24) { fill: orange;}",
+        "[id*=entity-Student] rect:nth-of-type(11), [id*=entity-Student] rect:nth-of-type(12), [id*=entity-Student] rect:nth-of-type(13) { fill: silver;}",
+        "[id*=entity-Candidate] rect:nth-of-type(21) { fill: orange;}",
+        "[id*=entity-Candidate] rect:nth-of-type(8), [id*=entity-Candidate] rect:nth-of-type(9), [id*=entity-Candidate] rect:nth-of-type(10) { fill: silver;}"
+    ]
+}
```

Comment:
Styling ER diagrams is a bit of a pain, but it does help.  Except this did not render correctly... oh, because I added the schema at the last minute (`edfi.`) and that broke the integration. I can fix that.

Reply:
Fixed.

Reply:
May need additional context on the styling when capturing notes here.